src/TerminalLogger.jl

```diff
@@ -2,8 +2,9 @@
     TerminalLogger(stream=stderr, min_level=$ProgressLevel; meta_formatter=default_metafmt,
                    show_limited=true, right_justify=0)
 
-Logger with formatting optimized for readability in a text console, for example
-interactive work with the Julia REPL.
+Logger with formatting optimized for interactive readability in a text console
+(for example, the Julia REPL). This is an enhanced version of the terminal
+logger `Logging.ConsoleLogger` which comes installed in the REPL by default.
```

Comment:
It looks like `ConsoleLogger` is used outside the `REPL` stdlib too?

```console
$ julia --startup-file=no -e "using Logging; @show typeof(current_logger())"
typeof(current_logger()) = ConsoleLogger
```

Reply:
True, it's the default which is applied in the stdlib `Logging.__init__()` (which in turn replaces a `CoreLogging.SimpleLogger` instance).

Reply:
Tweaked to fix this.